gulpfile.babel.js

```diff
@@ -121,7 +127,9 @@ gulp.task('dev', ['dev:doc', 'dev:static', 'dev:applystyles', 'dev:generate'], (
   gulp.watch('lib/styleguide.js', ['dev:generate']);
 });
 
-gulp.task('build', ['sass', 'js:app', 'js:vendor', 'html', 'assets']);
+gulp.task('build', ['clean-dist'], () => {
+  gulp.start(['sass', 'js:app', 'js:vendor', 'html', 'assets']);
```

Comment:
gulp.start? That's new to me - and even Gulp API documentation does not have it documented. I believe `run-sequence` is way to go instead (to ensure `clean-dist` runs before others): https://www.npmjs.com/package/run-sequence